charts/spire/values.yaml

```
@@ -1,6 +1,8 @@
 # You can enable config/features that affect all services here.
 global:
   spire:
+    # -- Set to the value of your clusters `kubeadm init --service-dns-domain` flag
+    clusterDomain: "cluster.local"
```

```suggestion
global:
  # -- Set to the value of your clusters `kubeadm init --service-dns-domain` flag
  clusterDomain: "cluster.local"

  spire:
```

This isn't a spire setting. Think its better of at root level.